website/blog/2023-04-14-aluminium.md

```diff
@@ -0,0 +1,343 @@
+---
+author: Katarzyna Marek
+title: Metals v0.11.12 - Aluminium
+authorURL: https://github.com/kasiaMarek
+authorImageURL: https://github.com/kasiaMarek.png
+---
+
+We're happy to announce the release of Metals v0.11.12, which 
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">86</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">72</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">17</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center"></td>
+  </tr>
+</tbody>
+</table>
+
+For full details: [https://github.com/scalameta/metals/milestone/_num_?closed=1]
+(https://github.com/scalameta/metals/milestone/_num_?closed=1)
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+
+- [Introduce Error Reports](#introduce-error-reports)
+- [Code action for converting dependencies from sbt to mill style](#code-action-for-converting-dependencies-from-sbt-to-mill-style)
+- [Extracting shared mtags without scalameta dependency](#extracting-shared-mtags-without-scalameta-dependency)
+- [Improvements to semantic highlighting](#improvements-to-semantic-highlighting)
+- [Discovering ignored tests in test explorer](#discovering-ignored-tests-in-test-explorer)
```

Comment:
I usually have a `Miscellaneous` for smaller fixes and features that don't require much explanation, I would keep it there and add any other smaller fixes